src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
+const main = document.querySelector('.tree');
+
+[...li].forEach(el => {
```

Comment:
Why do you need to spead to a new array here?
You could use `li.forEach` directly.

Reply:
it has been fixed, thank you.
right now I know that it is possible to iterate collection with forEach()